File: fullname-storage/src/main/java/ru/reeson2003/jaegerplaygroung/fullnamestorage/FullnameEntity.java
package ru.reeson2003.jaegerplaygroung.fullnamestorage;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "full_names")
public class FullnameEntity {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;

    private String firstname;

    private String lastname;

}

File: fullnameapi/src/main/kotlin/ru/reeson2003/jaegerplaygroung/fullnameapi/dao/FullNameEntity.kt
package ru.reeson2003.jaegerplaygroung.fullnameapi.dao

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "full_names")
class FullNameEntity(
    @Id val id: Long? = null,
    @Column(name = "firstname") val firstName: String? = null,
    @Column(name = "lastname") val lastName: String? = null
)
